INSTALL.md

```diff
@@ -13,43 +12,41 @@ cmake -DCMAKE_BUILD_TYPE=Debug -DCGAL_DIR=/path/to/cgal.git
 make
 ```
 
-in the case of the building of an example in debug mode.
-
-For more information head over to the [CGAL manual](https://doc.cgal.org/latest/Manual/general_intro.html).
-Note that this page describes the setting of CGAL as a sources release and, as such,
-files are organized in a slightly different way, see the [Layout of the CGAL Git Repository](README.md).
+in the case of building some CGAL-provided examples in debug mode.
 
+Note that although CGAL is a header-only library, some parts of it must link to several external libraries, such as GMP, MPFR, etc.
 
 Building a Program Using CGAL
 =============================
 
-To compile a program using CGAL, simply set `CGAL_DIR` to the location
-of the directory containing `CGALConfig.cmake` (for example the root 
-of the extracted source archive or the root of a git checkout).
-
-Here is an example of how to build in debug the examples from the 3D Triangulations package:
+If you wish to build a program that is not provided with CGAL and does not already have a `CMakeLists.txt`,
+you can trigger the creation of a basic `CmakeLists.txt` by calling the script [`cgal_create_cmake_script`](Scripts/scripts/cgal_create_cmake_script)
+found in `/path/to/cgal.git/Scripts/scripts/` at the root of your program directory.
 
 ``` {.bash}
- cd /path/to/cgal.git/Triangulation_3/examples/Triangulation_3
+ git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
+ cd /path/to/your/program
+ /path/to/cgal.git/Scripts/scripts/cgal_create_cmake_script
  mkdir -p build/debug
  cd build/debug
- cmake -DCGAL_DIR:PATH=/path/to/cgal.git ../..
- make
+ cmake -DCMAKE_BUILD_TYPE=Debug -DCGAL_DIR:PATH=/path/to/cgal.git ../..
+ make your_program
 ```
 
-If you are trying to build examples or tests that do not already have a `CMakeLists.txt`,
-you can trigger its creation by calling the script [`cgal_create_cmake_script`](Scripts/scripts/cgal_create_cmake_script)
-found in `/path/to/cgal.git/Scripts/scripts/` at the root of the example/test directory.
-Here is an example for the examples of the 2D Triangulation package:
+Since the basic `CmakeLists.txt` created by the script `cgal_create_cmake_script` cannot
```

Comment:
```suggestion
Since the basic `CmakeLists.txt` created by the script `cgal_create_cmake_script` cannot
```
```suggestion
Since the basic `CMakeLists.txt` created by the script `cgal_create_cmake_script` cannot
```